ui/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
+
+describe('workspace-project App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should display welcome message', () => {
+    page.navigateTo();
+    expect(page.getTitleText()).toEqual('Welcome to dragon!');
```

Comment:
Same comment.
Is this supposed to be hardcoded or replaced by the environment variable?

Reply:
These are the angular test cases. It not meant to be replaced by environment variables.